commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java

```diff
@@ -191,6 +191,14 @@ public class Web3SignerBaseCommand implements BaseConfig, Runnable {
       description = "Enable access logs (default: ${DEFAULT-VALUE})")
   private final Boolean accessLogsEnabled = false;
 
+  @Option(
+      names = "--Xmetadata-files-parallel-processing-enabled",
```

Comment:
Think this be namespaced under --key-store and also this parallel part only applies to the fetching and decrypting of the key not the loading of the config itself. So maybe ```--key-store-parallel-processing-enabled`